policyengine-client/craco.config.js

```diff
@@ -0,0 +1,22 @@
+const CracoLessPlugin = require("craco-less");
+
+module.exports = {
+  plugins: [
+    {
+        plugin: CracoLessPlugin,
+        options: {
+            lessLoaderOptions: {
+                lessOptions: {
+                    modifyVars: {
+                        "primary-color": "#002766",
```

Comment:
Could you try the color from our slides? `#3d85c6`

Reply:
Looks like this (ignoring the logo):
![image](https://user-images.githubusercontent.com/35577657/140089787-7f5b559e-25bd-470b-8da8-5f5e2cca9289.png)

Reply:
Since it looks a bit weird with the chart, can try `#2c6496`, which according to https://mdigi.tools/darken-color/#3d85c6 is 25% darker.

Reply:
OK, pending a few bugs (logo slightly too high, _Your plan_ cut off), how does this look? I quite like it:
![image](https://user-images.githubusercontent.com/35577657/140239242-620b814f-b79d-41f1-a67a-b4bc3a685ccd.png)

Reply:
Hmm looks a bit clashy to me. I think the two blues need some more contrast. Maybe lighten the chart color to `#5091cc`? That's 10% lighter per https://mdigi.tools/lighten-color/#3d85c6